CriteoPublisherSdk/Sources/DataConsent/CR_GdprVersion.m

```diff
@@ -20,9 +20,13 @@
 #import "CR_GdprVersion.h"
 #import "NSString+Tcf.h"
 
+NSUInteger const CR_CriteoTcfVendorID = 91;
```

Comment:
I suggest adding a comment that explains where `91` comes from as it's not trivial.

Reply:
Done.